go.mod

```diff
@@ -172,11 +172,9 @@ replace (
 	github.com/NYTimes/gziphandler => github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
 	github.com/PuerkitoBio/purell => github.com/PuerkitoBio/purell v1.1.1
 	github.com/PuerkitoBio/urlesc => github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
-	github.com/agnivade/levenshtein => github.com/agnivade/levenshtein v1.0.1
```

Comment:
curious that we have (transitive) deps that are not being vendored.

@liggitt is that expected?

Reply:
yeah, go.sum includes all transitive deps at the module level, go.mod pins minimum preferred version of those deps, vendor only includes the transitive packages needed to build/test